nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -266,6 +266,10 @@ class VanillaDataManagerConfig(InstantiateConfig):
     camera_optimizer: CameraOptimizerConfig = CameraOptimizerConfig()
     """Specifies the camera pose optimizer used during training. Helpful if poses are noisy, such as for data from
     Record3D."""
+    dataset_camera_scale_factor: float = 1.0
```

Comment:
Maybe `camera_res_scale_factor`. I worry this may be ambiguous with extrinsic scale.

Reply:
done